panther_utils/include/panther_utils/moving_average.hpp

```diff
@@ -0,0 +1,51 @@
+#ifndef PANTHER_UTILS_MOVING_AVERAGE_HPP_
+#define PANTHER_UTILS_MOVING_AVERAGE_HPP_
+
+#include <deque>
+
+namespace panther_utils
+{
+
+template <typename T>
+class MovingAverage
+{
+public:
+  MovingAverage(const unsigned window_size = 5, const T initial_value = T(0))
```

Comment:
```suggestion
  MovingAverage(const std::size_t window_size = 5, const T initial_value = T(0))
```
Maybe `std::size_t` will suit better in this case. In general this is `unsigned`, but in more suitable way